README.md

```diff
@@ -66,7 +69,7 @@ If you face some issues while installing and making Portal up in your local, hav
 Setup for developers (Windows)
 ------------------------------
 
-1. Make sure you have installed Python 3.6, make sure you the right one (32/64 bits). [Source](https://www.python.org/downloads/). During installation please pay attention to the following details :
+1. Make sure you have installed Python 3.6 or 3.7, make sure you the right one (32/64 bits). [Source](https://www.python.org/downloads/). During installation please pay attention to the following details :
 - Tick/Select Add Python 3.6 to PATH
```

Comment:
We can add here some detail here like if you choose Python 3.6 then Do this else Do that.

Reply:
Do you mean to expand on the `PATH`?

Reply:
I think Satya wants one or two line on Path only :D